tests/client/amo/components/TestAddonDetail.js

```diff
@@ -25,9 +27,11 @@ function render({ addon = fakeAddon, ...customProps } = {}) {
   };
 
   return findRenderedComponentWithType(renderIntoDocument(
-    <I18nProvider i18n={i18n}>
-      <AddonDetail {...props} />
-    </I18nProvider>
+    <Provider store={createStore(initialState)}>
```

Comment:
Because an add-on now uses the `AddonMoreInfo` component which has an AMO-relative `Link` component, this needed to be added so the tests wouldn't fail.